controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -439,26 +442,88 @@ func (r *ROSAControlPlaneReconciler) reconcileClusterVersion(rosaScope *scope.RO
 }
 
 func (r *ROSAControlPlaneReconciler) updateOCMCluster(rosaScope *scope.ROSAControlPlaneScope, ocmClient *ocm.Client, cluster *cmv1.Cluster, creator *rosaaws.Creator) error {
+	ocmClusterSpec, updated := r.updateOCMClusterSpec(rosaScope.ControlPlane, cluster)
+
+	if updated {
+		// Update the cluster.
+		rosaScope.Info("Updating cluster")
+		if err := ocmClient.UpdateCluster(cluster.ID(), creator, ocmClusterSpec); err != nil {
+			conditions.MarkFalse(rosaScope.ControlPlane,
+				rosacontrolplanev1.ROSAControlPlaneValidCondition,
+				rosacontrolplanev1.ROSAControlPlaneInvalidConfigurationReason,
+				clusterv1.ConditionSeverityError,
+				err.Error())
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (r *ROSAControlPlaneReconciler) updateOCMClusterSpec(rosaControlPlane *rosacontrolplanev1.ROSAControlPlane, cluster *cmv1.Cluster) (ocm.Spec, bool) {
```

Comment:
```suggestion
func  updateOCMClusterSpec(rosaControlPlane *rosacontrolplanev1.ROSAControlPlane, cluster *cmv1.Cluster) (ocm.Spec, bool) {
```
this can just be a function, you are not using any field from `ROSAControlPlaneReconciler`